### PR TITLE
CT-1849 - bucket lastShareChangeDate update test

### PR DESCRIPTION
**Pull request by vojtabiberle**
Jira: [CT-1849](https://keboola.atlassian.net/browse/CT-1849)
KBC: https://github.com/keboola/connection/pull/5363

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] New client method(s) support branches, or they are listed in BranchAwareClient 
- [x] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

[CT-1849]: https://keboola.atlassian.net/browse/CT-1849?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tests/Backend/CommonPart1/BucketsTest.php [hunk 1]
```diff
@@ -640,6 +642,51 @@ public function testBucketOwner(string $devBranchType, string $userRole): void
         $bucket = $this->_testClient->getBucket($bucketId);
         $this->assertEquals($ownerId, $bucket['owner']['id']);
         $this->assertEquals($ownerName, $bucket['owner']['name']);
+        $this->assertNotNull($bucket['lastShareChangeDate']);
+    }
+
+    /**
+     * @dataProvider provideComponentsClientTypeBasedOnSuite
+     */
+    public function testBucketDescriptionMetadata(): void
```
Comment (romanbracinik) [inline tests/Backend/CommonPart1/BucketsTest.php:651]: asi by som rad videl testy kde udatujes napr displayName ale `lastShareChangeDate` sa neupdatne. 
Plus by som ties otestoval asi ze sa to nastavi pri share akciach ked budes uz upravovat toto.

**Review by romanbracinik — CHANGES_REQUESTED** [review]
Pridavas to ten novy param aj do response tak by to malo byt aj v doc

File: tests/Backend/CommonPart1/BucketsTest.php (hunk 1 above)
Comment (vojtabiberle) [reply tests/Backend/CommonPart1/BucketsTest.php:651]: good point, dodělám

Comment (vojtabiberle) [reply tests/Backend/CommonPart1/BucketsTest.php]: done, i s aktualizovanou dokumentací

**Review by romanbracinik — APPROVED** [review]
super za mna tato ok

**Pull request merged**